src/__experimental__/components/switch/switch.stories.js

```diff
@@ -67,31 +67,58 @@ const switchComponent = () => switchWrapper({
   store: formStore
 });
 
-const switchComponentValidation = () => (
-  <>
-    {validationTypes.map(type => switchWrapper({
-      ...commonKnobs(),
-      ...dlsKnobs(),
-      key: `key-${type}`,
-      name: `switch-${type}`,
-      label: text(`label ${type}`, `Please read our ${type}`),
-      value: type,
-      store: stores[type],
-      onChange: handleChange(stores[type]),
-      validations: testValidation('valid'),
-      warnings: testValidation('warn'),
-      info: testValidation('info'),
-      unblockValidation: trueBool,
-      useValidationIcon: trueBool
-    }))}
-  </>
-);
+const switchComponentValidation = (props = { excludeKnobs: false }) => {
+  const { excludeKnobs } = props;
+  const knobs = excludeKnobs ? {} : { ...commonKnobs(), ...dlsKnobs() };
+  return (
+    <>
+      {validationTypes.map(type => switchWrapper({
+        ...knobs,
+        key: `key-${type}`,
+        name: `switch-${type}`,
+        label: text(`label ${type}`, `Please read our ${type}`),
+        value: type,
+        store: stores[type],
+        onChange: handleChange(stores[type]),
+        validations: testValidation('valid'),
+        warnings: testValidation('warn'),
+        info: testValidation('info'),
+        unblockValidation: trueBool,
+        useValidationIcon: trueBool
+      }))}
+    </>
+  );
+};
 
 storiesOf('Experimental/Switch', module)
-  .add(...makeStory('default', dlsThemeSelector, switchComponent))
-  .add(...makeStory('classic', classicThemeSelector, switchClassic))
-  .add(...makeStory('validations', dlsThemeSelector, switchComponentValidation))
-  .add(...makeStory('validations classic', classicThemeSelector, switchComponentValidation));
+  .add(
```

Comment:
the last add( was going over the line limit, I thought this would be best go do fo consistency.
I can try to do this I guess, unless you have better idea?

`.add(...makeStory('validations classic', classicThemeSelector, () => switchComponentValidation(
    { excludeKnobs: true })
  ));`

Reply:
Oh line limit... :) So let's stay with the current approach